src/ambianic/pipeline/ai/fall_detect.py

```diff
@@ -126,15 +127,21 @@ def find_keypoints(self, image):
         # if no pose detected with high confidence, 
         # try rotating the image +/- 90' to find a fallen person
         # currently only looking at pose[0] because we are focused on a lone person falls
-        while (not poses or poses[0].score < min_score) and rotations:
+        #while (not poses or poses[0].score < min_score) and rotations:
+        pose_score, _ = self.estimate_spinalVector_score(poses[0])
+        while pose_score < min_score and rotations:
           angle = rotations.pop()
           transposed = image.transpose(angle)
           # we are interested in the poses but not the rotated thumbnail
           poses, _ = self._pose_engine.DetectPosesInImage(transposed)
+          pose_score, _ = self.estimate_spinalVector_score(poses[0])
+
         if poses and poses[0]:
             pose = poses[0]
+            pose_score, _ = self.estimate_spinalVector_score(pose)
```

Comment:
@bhavikapanara why do we calculate the same pose_score twice? We already have the latest pose_score from the while loop above. We can also keep a current pose reference along the current pose_score reference and reuse both to avoid the redundant code and CPU usage.

Reply:
Yes, @ivelin  you are correct...it is redundant call.